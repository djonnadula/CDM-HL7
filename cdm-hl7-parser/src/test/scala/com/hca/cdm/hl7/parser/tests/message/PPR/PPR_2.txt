MSH|^~\&|COCQA1A|COCQA1A|OV|OV|201611091511||PPR^PC1|IP_COQ1A_PPR_EPSQ1A.161208.1867|T|2.5||KYA^04446|ER|NE
PID|1||J000425961^^^COCQA1A^MRN|J502566|PARKER^MAY^R||19620105|F
PRB|AD|201611091510^1TSQBE8554^DRBRI|161891005^Back pain^SNOMED|J502566-3|||201611091510^1TSQBE8554^DRBRI|||Resolved||Resolved||||20161031