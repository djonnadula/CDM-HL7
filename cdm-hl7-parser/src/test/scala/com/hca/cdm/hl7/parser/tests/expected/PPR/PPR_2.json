{
  "0000.COMN": {
    "sending_facility": "COCQA1A",
    "message_control_id": "IP_COQ1A_PPR_EPSQ1A.161208.1867",
    "medical_record_num": "J000425961",
    "medical_record_urn": "J502566",
    "patient_account_num": ""
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "COCQA1A",
    "sending_facility": "COCQA1A",
    "receiving_application": "OV",
    "receiving_facility": "OV",
    "date_time_of_message": "201611091511",
    "message_type": {
      "message_code": "PPR",
      "trigger_event": "PC1"
    },
    "message_control_id": "IP_COQ1A_PPR_EPSQ1A.161208.1867",
    "processing_id": "T",
    "version_id": "2.5",
    "continuation_pointer": {
      "network_id": "KYA",
      "unit_number": "04446"
    },
    "accept_acknowledgement_type": "ER",
    "application_acknowledgement_type": "NE"
  },
  "0002.PID": {
    "set_id": "1",
    "patient_id_list": {
      "id_num": "J000425961",
      "assigning_authority": "COCQA1A",
      "id_type_code": "MRN"
    },
    "alternate_patient_id": "J502566",
    "patient_name": {
      "family_name": "PARKER",
      "given_name": "MAY",
      "second_and_further_given_names_or_initials_thereof": "R"
    },
    "date_time_of_birth": "19620105",
    "administrative_sex": "F"
  },
  "0003.PRB": {
    "action_code": "AD",
    "action_date_time": {
      "date_time": "201611091510",
      "person_id": "1TSQBE8554",
      "provider_id": "DRBRI"
    },
    "problem_id": {
      "primary_id": "161891005",
      "text": "Back pain",
      "name_of_coding_sys": "SNOMED"
    },
    "problem_instance_id": "J502566-3",
    "problem_estbl_date_time": {
      "date_time": "201611091510",
      "person_id": "1TSQBE8554",
      "provider_id": "DRBRI"
    },
    "problem_classification": "Resolved",
    "problem_persistence": "Resolved",
    "problem_date_of_onset": "20161031"
  }
}